tests/nlu/test_evaluation.py

```diff
@@ -1219,3 +1219,25 @@ def test_replacing_fallback_intent():
 def test_is_entity_extractor_present(components, expected_result):
     interpreter = Interpreter(components, context=None)
     assert is_entity_extractor_present(interpreter) == expected_result
+
+
+def test_plot_attribute_confidences_with_invalid_values(tmp_path: Path):
+    path = tmp_path / "evaluation"
+    path.mkdir()
+    report_folder = str(path / "reports")
+
+    rasa.shared.utils.io.create_directory(report_folder)
+
+    intent_results = [
+        IntentEvaluationResult("", "restaurant_search", 76, -21),
+        IntentEvaluationResult("greet", "greet", -32, 99),
+    ]
+
+    with pytest.raises(InvalidTypeException):
+        evaluate_intents(
```

Comment:
I noticed in the Sentry report that the error was triggered by a call to `evaluate_response_selections` function, what about creating a separate test for this too?

Reply:
Very good catch on seeing that it started from there ! 💡 It looks though that if I call `evaluate_intents` or `evaluate_response_selections`, it does the same thing in a test related level. It arrives at the function `plot_attribute_confidences`, checks that it has an integer in `pos_hist` or `neg_hist` and raises an exception. I would not add a separate test for this but I can replace the current test by using `evaluate_response_selections`.

Reply:
Sounds good 👍